awscli/clidocs.py

```diff
@@ -159,6 +159,7 @@ def doc_option(self, arg_name, help_command, **kwargs):
             name = '``%s``' % argument.cli_name
         doc.write('%s (%s)\n' % (name, argument.cli_type_name))
         doc.style.indent()
+        self._document_enums(argument, doc)
```

Comment:
Yeah I agree with that.  Seems like the AWS API reference docs put the enums/metadata after the doc strings (e.g http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html)